### PR TITLE
fix(api-v3): fix return types for a few functions

### PR DESCRIPTION
**Pull request by Halen84**
noticed some functions were not returning the correct type. also added code comments for clarity.

### CODE REVIEW

File: source/scripting_v3/GTA/Entities/Peds/Ped.cs [hunk 1]
```diff
@@ -962,9 +962,9 @@ private int GetEventTypeIndexForB1737OrOlder(EventType eventType)
 		#region Euphoria & Ragdoll
 
 		/// <inheritdoc cref="Ragdoll(int, RagdollType, bool)"/>
-		public void Ragdoll(int duration = -1, RagdollType ragdollType = RagdollType.Relax)
```
Comment (kagikn) [inline source/scripting_v3/GTA/Entities/Peds/Ped.cs:965]: This signature is shipped in the stable release v3.6.0. Although the other one is not shipped in it, we cannot change the return type of this signature.

File: source/scripting_v3/GTA/Entities/Vehicles/Vehicle.cs [hunk 2]
```diff
@@ -2111,9 +2111,10 @@ public void TowVehicle(Vehicle vehicle, bool rear)
 		/// Detaches specified vehicle from any tow truck it might be attached through, loops through all vehicles so could be expensive.
 		/// If you know the tow truck <see cref="Vehicle"/> that tows this <see cref="Vehicle"/>, you should call <see cref="DetachTowedVehicle"/> on the tow truck.
 		/// </summary>
-		public void DetachFromTowTruck()
```
Comment (kagikn) [inline source/scripting_v3/GTA/Entities/Vehicles/Vehicle.cs:2114]: Shipped in the stable release v3.6.0.

**Review by kagikn — CHANGES_REQUESTED** [review]
I appreciate the return type correction for signatures that are not shipped in any stable versions! This PR saves my time to check if something is wrong with how to use native functions for new members (methods or properties) before shipping v3.7.0.

We allow breaking changes for signatures that are not shipped in any stable versions. However, there are 2 signatures that are shipped in stable versions and those changes will result in `MissingMethodException`s. It figures (to me), though, since things will be different around the stack and registers.

Binary compatibility does matter more than correcting return types if target signatures are shipped in some stable releases (unless in first version in a major version).

**Comment by Halen84** [comment]
ah makes sense. ill push a fix real quick. thanks.

**Comment by kagikn** [comment]
You may want to see [the Breaking Change Rules page](https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/breaking-change-rules.md) in .NET runtime repo so you can grasp what changes will break binary or source compatibility.

**Comment by Halen84** [comment]
Thanks, I'll be sure to check that out next time. Not sure how many PR's I'll make to this repo, I was just merging our codebases and just noticed those and thought I'd fix them real quick.

**Review by kagikn — APPROVED** [review]
Looking good now.

**Pull request merged**